src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +53,36 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User bob = new User();
+        bob.setEmail("bob@gmail.com");
+        bob.setPassword("12345");
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+        try {
+            System.out.println(authenticationService.register(bob.getEmail(), bob.getPassword()));
+            System.out.println(authenticationService.login(bob.getEmail(), bob.getPassword()));
+        } catch (RegistrationException | AuthenticationException e) {
+            System.out.println("This should work");
+        }
+        try {
+            System.out.println(authenticationService.register(bob.getEmail(), bob.getPassword()));
+        } catch (RegistrationException e) {
+            System.out.println("Registration checker work correctly");
+        }
+        try {
+            System.out.println(authenticationService.login(bob.getEmail(), bob.getPassword()));
+        } catch (AuthenticationException e) {
+            System.out.println("Login checker work correctly");
+        }
```

Comment:
The login attempt should verify that the `AuthenticationService` correctly handles authentication logic. Ensure that the service is implemented to throw an `AuthenticationException` if login credentials are incorrect.